tasks/src/text-generation/data.ts

```diff
@@ -101,7 +105,7 @@ const taskData: TaskDataCustom = {
 		},
 	],
 	summary:      "Generating text is the task of producing new text. These models can, for example, fill in incomplete text or paraphrase.",
-	widgetModels: ["gpt2"],
+	widgetModels: ["bigscience/bloom-560m"],
```

Comment:
I think we can use falcon here

Reply:
I used instruction fine-tuned 7B version as for the other ones widget was turned off (it was reason why I didn't pick it as well :/)